addons/web/tests/test_domain.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+from odoo.tools.misc import mute_logger
+
+class DomainTest(HttpCase):
+
+    def test_domain_validate(self):
+        self.authenticate('admin', 'admin')
+
+        with mute_logger('odoo.http'):
+            resp = self.url_open('/web/domain/validate',
+                headers={'Content-Type': 'application/json'},
+                data=json.dumps({'params':{'model':'i', 'domain':[], }})
+            )
```

Comment:
Just a style suggestion
```suggestion
            resp = self.url_open(
                '/web/domain/validate',
                headers={'Content-Type': 'application/json'},
                data=json.dumps({'params': {'model':'i', 'domain':[]}}),
            )
```